storybook/src/components/Button.js

```diff
@@ -0,0 +1,43 @@
+import React, {PureComponent} from 'react';
+import {CSSTransition} from 'react-transition-group';
+import "../css/Button.css";
+import classNames from 'classnames';
+
+export class Button extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            label: props.label,
+            round: props.round,
+            classNames: "custom-button ",
+            id: this.props.id,
```

Comment:
Why did you choose to include id in the state if it gets it from the props? What if the props change, what will happen to the state?